test/e2e/wallet-helpers/api-helpers.ts

```diff
@@ -0,0 +1,60 @@
+import { WebDriver } from 'selenium-webdriver'
+export class APIHelper {
+  private driver: WebDriver
+
+  constructor(driver: WebDriver) {
+    this.driver = driver
+  }
+
+  async generateRecoveryPhrase() {
+    return await this.driver.executeScript<string>(async () => {
+      // @ts-ignore
+      const { recoveryPhrase } = await window.client.request('admin.generate_recovery_phrase', null)
+      return recoveryPhrase
+    })
+  }
+
+  async importWallet(recoveryPhrase: string) {
+    return (
+      await this.driver.executeScript<string>(async (recoveryPhrase: string) => {
+        // @ts-ignore
+        const resp = await window.client.request('admin.import_wallet', {
```

Comment:
thought (non-blocking): having magic strings for method names is a bit suboptimal. Unfortunately because of scope/import issues we experienced you cannot import a map or anything here. I think out of scope for this PR but would like to see these shared between tests and FE code eventually.

Reply:
Yup, love this